### PR TITLE
add speed

### LINKED ISSUE #8: Increase the speed with the player's score
The more the score is high, the faster the game could be

### PR DESCRIPTION
**Pull request by Blimy**
Goal here is to inscrease the speed of the snake as the global score is increasing so we kind of introduce a notion of "level". I arbitrary decided that the player will see the speed of the snake increasing every 5 points. 
It's important to take into account thant the speed will never be less than 30ms (for the update of snake position, and so the visual speed of it) as it will not be "playable" if it is inferior to this value. 
So to sum up the modification: 
- starting speed is 80 ms
- every 5 points the speed increasing (so time refresh is descreasing by 10 ms)
- when time refresh should be inferior to 30ms it stays at 30 ms

Closes #8

### CODE REVIEW

File: js/snake.js [hunk 1]
```diff
@@ -171,8 +174,8 @@ class Game {
         // Draw the food
         var food = new Image();
         food.src = "js/Phone.jpg";
-        
-        ctx.drawImage(food, this.food[0] * this.tileWidth - food.width/4, this.food[1] * this.tileHeight - food.height/4);
+        //TO DO : Randomize placement
+        ctx.drawImage(food, this.food[0] * this.tileWidth, this.food[1] * this.tileHeight)
```
Comment (phsym) [inline js/snake.js]: ???

File: js/snake.js [hunk 2]
```diff
@@ -214,13 +217,31 @@ function start() {
...
+		// if speed should be faster to 30 ms, keep it to 30 ms, so the player can still play
+		if(snakeFastIndex>5){
+			var speed = 30;
+		}
+		else{
+			var speed = 80 - (10*snakeFastIndex);
+		}
+	}
+	console.log("snake speed = "+speed);
+	timer = setInterval(loop, speed, game, ctx);
```
Comment (phsym) [inline js/snake.js]: I think you want to use `setTimeout` here instead of `setInterval`, don't you ?

Comment (Blimy) [reply js/snake.js]: True. Will commit new changes containing this
